airflow/pipe_vessels_dag.py

```diff
@@ -19,53 +22,71 @@ def get_dag_id(self, prefix, vessel_pipe_name):
         return '{}.{}'.format(prefix, vessel_pipe_name)
 
     def build(self, dag_id):
-        
+
         config = self.config
         config.update(self.vessel_config)
 
         with DAG(dag_id, schedule_interval=self.schedule_interval, default_args=self.default_args) as dag:
-           
+
             config['date_range'] = ','.join(self.source_date_range())
             name = config['name']
+            flexible_operator_var = Variable.get('FLEXIBLE_OPERATOR')
+
+            publish_vessel_info_params = {
+                'task_id':'publish_vessel_info_{}'.format(name),
+                'depends_on_past':True,
+                'pool':'k8operators_limit' if flexible_operator_var == 'kubernetes' else 'bigquery',
+                'docker_run':'{docker_run}'.format(**config),
+                'image':'{docker_image}'.format(**config),
+                'name':'pipe-vessels-publish-vessel-info-{}'.format(name),
+                'dag':dag,
+                'arguments':['publish_vessel_info',
+                             '\'{bigquery_vessel_info_query}\''.format(**config),
+                             '{project_id}:{temp_dataset}'.format(**config),
+                             '{temp_bucket}'.format(**config),
+                             '{elasticsearch_server_url}'.format(**config),
+                             '{elasticsearch_server_auth}'.format(**config),
+                             '{elasticsearch_index_alias}'.format(**config),
+                             '\'{elasticsearch_index_mappings}\''.format(**config)]
+            }
+            publish_vessel_info = FlexibleOperator(publish_vessel_info_params).build_operator(flexible_operator_var)
+
+            aggregate_tracks_params = {
+                'task_id':'aggregate_tracks_{}'.format(name),
+                'depends_on_past':True,
+                'pool':'k8operators_limit' if flexible_operator_var == 'kubernetes' else 'bigquery',
+                'docker_run':'{docker_run}'.format(**config),
+                'image':'{docker_image}'.format(**config),
+                'name':'pipe-vessels-aggregate-tracks-{}'.format(name),
+                'dag':dag,
+                'arguments':['aggregate_tracks',
+                             '{date_range}'.format(**config),
+                             '\'{bigquery_vessel_tracks_jinja_query}\''.format(**config),
+                             '{project_id}:{target_dataset}.{bigquery_tracks}'.format(**config)]
+            }
+            aggregate_tracks = FlexibleOperator(aggregate_tracks_params).build_operator(flexible_operator_var)
+
+            publish_postgres_tracks_params = {
+                'task_id':'publish_postgres_tracks_{}'.format(name),
+                'depends_on_past':True,
+                'pool':'k8operators_limit',
```

Comment:
Shouldn't this be:
`                'pool':'k8operators_limit' if flexible_operator_var == 'kubernetes' else '',`